### PR TITLE
Dcompact for iris

### PR DESCRIPTION
**Pull request by loksonarius**
## Description

Adding a new implementation of my keymap (dcompact) for the Iris keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/iris/keymaps/dcompact/keymap.c [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    case LOWER:
+      if (record->event.pressed) {
+        layer_on(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/dcompact/keymap.c:163]:
```suggestion
```

File: keyboards/keebio/iris/keymaps/dcompact/keymap.c [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+    case RAISE:
+      if (record->event.pressed) {
+        layer_on(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/dcompact/keymap.c:173]:
```suggestion
```

File: keyboards/keebio/iris/keymaps/dcompact/keymap.c [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+    case FUNC:
+      if (record->event.pressed) {
+        layer_on(_FUNC);
+      } else {
+        layer_off(_FUNC);
+      }
+      return false;
+      break;
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/dcompact/keymap.c:181]:
```suggestion
```

File: keyboards/keebio/iris/keymaps/dcompact/keymap.c [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+       if (!record->event.pressed) {
+         #ifdef AUDIO_ENABLE
+           stop_all_notes();
+           PLAY_SONG(plover_song);
+         #endif
+         rgblight_mode(RGBLIGHT_MODE_SNAKE);
+         layer_on(_GAME);
+       }
+       return false;
+       break;
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/dcompact/keymap.c:192]:
```suggestion
```

File: keyboards/keebio/iris/keymaps/dcompact/keymap.c [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+     case EXT_GM:
+       if (record->event.pressed) {
+         #ifdef AUDIO_ENABLE
+           PLAY_SONG(plover_gb_song);
+         #endif
+         rgblight_mode(RGBLIGHT_MODE_RAINBOW_MOOD);
+         layer_off(_GAME);
+       }
+       return false;
+       break;
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/dcompact/keymap.c:202]:
```suggestion
```

File: keyboards/keebio/iris/keymaps/dcompact/keymap.c [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+     case LNUM:
+       if (record->event.pressed) {
+         layer_on(_LNUM);
+       } else {
+         layer_off(_LNUM);
+       }
+       return false;
+       break;
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/dcompact/keymap.c:210]:
```suggestion
```

File: keyboards/keebio/iris/keymaps/dcompact/keymap.c [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+     case LFUN:
+       if (record->event.pressed) {
+         layer_on(_LFUN);
+       } else {
+         layer_off(_LFUN);
+       }
+       return false;
+       break;
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/dcompact/keymap.c:218]:
```suggestion
```

**Comment by loksonarius** [comment]
@fauxpark removed the trailing `break`s from the switch statements

**Pull request merged**